src/Symfony/Bundle/FrameworkBundle/Resources/config/property_access.xml

```diff
@@ -4,10 +4,22 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="property_accessor.cache.prefix" />
+    </parameters>
+
     <services>
         <service id="property_accessor" class="Symfony\Component\PropertyAccess\PropertyAccessor" >
             <argument /> <!-- magicCall, set by the extension -->
             <argument /> <!-- throwExceptionOnInvalidIndex, set by the extension -->
+            <argument /> <!-- cache, set by the extension -->
+        </service>
+
+        <!-- Cache -->
+        <service id="property_accessor.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
```

Comment:
Yes it needs the BC layer (btw supporting APCu in Doctrine Cache should be easy). But ok to switch to the filesystem cache by default, as you said in the other thread it works almost everywhere (except for RO filesystems but they already require some tweaks).

As APC support is shipped (and used) with 2.8/3.0, we have to maintain it until 4.0 right? So we add a new service for the filesystem cache everywhere (validator, serializer, property info at least) and configure it as default service while keeping APC for BC (except here as it is new)?